src/lib/localStorage.js

```diff
@@ -1,13 +1,8 @@
 export function updateLocalStorage(key, data) {
-    localStorage.setItem(key, JSON.stringify(data));
-  }
-  
-  export function loadFromLocalStorage(key) {
-    try {
-      const localData = localStorage.getItem(key);
-      return JSON.parse(localData);
-    } catch (error) {
-      console.error(error);
-    }
-  }
-  
\ No newline at end of file
+  localStorage.setItem(key, JSON.stringify(data));
+}
+
+export function loadFromLocalStorage(key) {
+  const data = JSON.parse(localStorage.getItem(key)) || [];
```

Comment:
any reason why you deleted the error handling here?

Reply:
it's postponed until next week